code/frontend/src/pages/chat/Chat.tsx

```diff
@@ -168,12 +167,12 @@ const Chat = () => {
           "Azure Speech subscription key or region is not defined."
         );
       } else {
-        const speechConfig = SpeechConfig.fromSubscription(
-          subscriptionKey,
-          serviceRegion
-        );
-        const audioConfig = AudioConfig.fromDefaultMicrophoneInput();
-        const recognizer = new SpeechRecognizer(speechConfig, audioConfig);
+        const recognizer = multiLingualSpeechRecognizer(subscriptionKey, serviceRegion, [
+          "en-US",
+          "fr-FR",
+          "de-DE",
+          "it-IT"
+        ]);
```

Comment:
This file is doing too much, and has many many imports. I needed to test the multi-lingual configuration for the speech recognizer, so I extracted it into another file, and tested that file. As we're adding new functionality to CWYD, we should try and modularize this code so that each file is of a manageable size and complexity and is conducive for testing.